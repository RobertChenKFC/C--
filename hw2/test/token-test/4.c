../assignment-test/4.c