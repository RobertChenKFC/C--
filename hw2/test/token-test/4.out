IDENTIFIER
IDENTIFIER
IDENTIFIER
COMMENT
/****  start ***/
[Error] Lexer: unrecognized token ~ on line 6
